pkg/cvo/cvo.go

```diff
@@ -655,7 +655,7 @@ func (optr *Operator) availableUpdatesSync(ctx context.Context, key string) erro
 // sync upgradeableCondition. It only modifies cluster version.
 func (optr *Operator) upgradeableSync(ctx context.Context, key string) error {
 	startTime := time.Now()
-	klog.V(2).Infof("Started syncing upgradeable %q (%v)", key, startTime)
+	klog.V(2).Infof("Started syncing upgradeable %q (%v)", key, startTime.Format(time.RFC3339))
```

Comment:
curious since the log itself have time so why ever printing time.Now() in the first place? the result is a duplicate like so: (from old cvo olg)
```
I0110 06:11:27.263590       1 cvo.go:607] Started syncing upgradeable 
 "openshift-cluster-version/version" (2023-01-10 06:11:27.263587036 +0000 UTC m=+161.666330195)
```

Reply:
That's a good point, let's remove these redundancies
/hold